### DISCUSSION TITLE
What should I do on the client side?

### DISCUSSION THREAD

**Discussion by Saya47** [angristan/wireguard-install#403] [Help]
Hello! I install the script on the server, and it created a new client and the end for me, it's called `wg0-client-c1.conf`. The README doesn't say anything about the client side, so I thought I should install the same script on the client as well but copy/paste the content of the client config file to this location: `/etc/wireguard/wg0.conf`, the systemd service is OK, and the `ip a` shows this: 
```
6: wg0: <POINTOPOINT,NOARP,UP,LOWER_UP> mtu 1420 qdisc noqueue state UNKNOWN group default qlen 1000
    link/none
    inet 10.66.66.2/32 scope global wg0
       valid_lft forever preferred_lft forever
    inet6 fd42:42:42::2/128 scope global
       valid_lft forever preferred_lft forever
```
And `ip r get 1.1.1.1` shows the `dev wg0` being used. But I don't have internet access! Pinging any address fails!

As a side question, can I access the services running on the server from the client side? For example if the server's IP is 2.2.2.2 how can I access a service that's running on port 222 for example? Also, is it possible for the client to access the services that are running on the loopback interface of the server?

Thanks very much in advance.

**Comment by emadbagheri96** [comment]
Hello.
This script is only for running a VPN server.
On the client side you could use this link for all OSs and then use the `.conf` file in them:
[https://www.wireguard.com/install/](https://www.wireguard.com/install/)
